core/cas-server-core-authentication-api/src/test/resources/GroovyAuthenticationPolicy.groovy

```diff
@@ -0,0 +1,4 @@
+def shouldResumeOnFailure(Object[] args) {
```

Comment:
This is not a valid authentication policy file. Please update the script to match the docs.

Reply:
Hi,
No problem, but I have copied this file from the [GroovyScriptAuthenticationPolicyTests.java](https://github.com/apereo/cas/blob/master/core/cas-server-core-authentication-api/src/test/java/org/apereo/cas/authentication/policy/GroovyScriptAuthenticationPolicyTests.java) examples. Should I also update them ?

Reply:
No. You should only update the script file.

Reply:
Done, the test file is now a copy from [the documentation example](https://apereo.github.io/cas/6.5.x/authentication/Configuring-Authentication-Policy-Groovy.html).